test/Microsoft.AspNetCore.AzureAppServices.FunctionalTests/ProcessExtensions.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.AzureAppServices.FunctionalTests
+{
+    internal static class ProcessExtensions
+    {
+        private static readonly TimeSpan _defaultTimeout = TimeSpan.FromSeconds(30);
```

Comment:
This is copy paste from https://github.com/aspnet/Common/blob/dev/shared/Microsoft.Extensions.Process.Sources/ProcessHelper.cs use the sources package